### DISCUSSION TITLE
Allow to be notified of releases, but not pre-releases

### DISCUSSION THREAD

**Discussion by diktomat** [community/community#44125] [Repositories]
### Select Topic Area

Product Feedback

### Body

I like to be notified of new releases of projects that are just not there yet, but some of them do nightly pre-releases, which are kind of annoying to see in my inbox every day. Would be great if one could watch a repo for releases **excluding pre-releases**.

**Comment by m4heshd** [comment]
It would be amazing to have this feature.
